mesonbuild/backend/backends.py

```diff
@@ -1475,7 +1475,7 @@ def create_install_data(self) -> InstallData:
         strip_bin = self.environment.lookup_binary_entry(MachineChoice.HOST, 'strip')
         if strip_bin is None:
             if self.environment.is_cross_build():
-                mlog.warning('Cross file does not specify strip binary, result will not be stripped.')
+                mlog.warning('Cross file does not specify strip binary, result will not be stripped.', fatal=False)
```

Comment:
Tbh this is a bad cross file configuration which causes your configure settings to be ignored, it's absolutely a user thing to fix.

Reply:
I'm not sure I agree, I never specify strip in my cross files because I simply don't care about stripping them (I'm basically just compile testing, I'm not actually going to use them). but It's easy enough for someone to fix if they want fatal warnings I guess. Dropped